makemkv/makemkv.py

```diff
@@ -237,9 +238,15 @@ def _translate_codes(
                 # convert 3-letter language codes to 2-letter codes
                 value = Lang(value).pt1
 
+            if key == "framerate":
+                # convert "##.### (#####/####)" to int string
+                if re.search("^\d+\.\d*\s\(\d+/\d+\)$", value):
```

Comment:
Quick note: this needs to be a raw string (`r""`).

Reply:
Thanks for all the suggestions. I'll fix them as soon as possible